core/plugin_registry.go

```diff
@@ -44,7 +44,8 @@ import (
 func hasAWSCredentials() bool {
 	if os.Getenv("AWS_PROFILE") != "" ||
 		os.Getenv("AWS_ROLE_SESSION_NAME") != "" ||
-		(os.Getenv("AWS_ACCESS_KEY_ID") != "" && os.Getenv("AWS_SECRET_ACCESS_KEY") != "") {
+		(os.Getenv("AWS_ACCESS_KEY_ID") != "" && os.Getenv("AWS_SECRET_ACCESS_KEY") != "" && os.Getenv("AWS_DEFAULT_REGION") != "") {
```

Comment:
This seems incorrect.

- AWS credentials (access key ID and secret access key) are authentication-related
- AWS region is a configuration parameter, not a credential
- Many AWS SDKs can function without `AWS_DEFAULT_REGION` by using service-specific endpoints or falling back to default regions

Reply:
Maybe using AddSetupQuestion if the region cant be read would be a better solution?